### DISCUSSION TITLE
OKD SNC installation issue

### DISCUSSION THREAD

**Discussion by Prakash1977** [okd-project/okd#1927] [General]
when i run this below command 

openshift-install --dir=${OKD4_SNC_PATH}/okd4-install-dir wait-for bootstrap-complete --log-level debug

Getting below

DEBUG Still waiting for the Kubernetes API: Get "https://api.okd4-snc.snc.test:6443/version": dial tcp 192.168.38.176:6443: connect: connection refused

**Comment by JaimeMagiera** [comment]
You are attempting single node? Which instructions are you following?
